src/page-templates/topic-content-type-page/topic-content-type-page-template.tsx

```diff
@@ -246,25 +283,53 @@ export const TopicContentTypePageTemplate: NextPage<
                             }}
                         />
                     )}
-                    <Search
-                        titleElement="h1"
-                        title={`All ${topicName} ${pluralize(contentType)}`}
-                        placeholder={`Search ${topicName} ${pluralize(
-                            contentType
-                        )}`}
-                        tagSlug={topicSlug}
-                        contentType={contentType}
-                        pageNumber={pageNumber}
-                        pageSlug={(topicSlug + contentTypeSlug).split('/')}
-                        setSeoAttributes={setSeoAttributes}
-                        initialSearchContent={initialSearchContent}
-                        resultsLayout="grid"
-                        titleLink={getSearchTitleLink(
-                            contentType,
-                            contentTypeAggregateSlug
+
+                    <div sx={extraSearchWrapperStyles}>
+                        <div sx={titleStyles}>
+                            <TypographyScale
+                                variant="heading5"
+                                customElement="h5"
+                            >
+                                All {topicName} {pluralize(contentType)}
+                            </TypographyScale>
+
+                            <ExpandingLink
+                                {...getSearchTitleLink(
+                                    contentType,
+                                    contentTypeAggregateSlug
+                                )}
+                                hoverStyleOverrides={linkStyleOverride}
+                            />
+                        </div>
+
+                        <SearchBox
+                            {...searchBoxProps}
+                            placeholder={`Search ${topicName} ${pluralize(
+                                contentType
+                            )}`}
+                            extraStyles={extraSearchBoxStyles}
+                        />
+                        <SortBox
+                            {...sortBoxProps}
+                            extraStyles={extraSortBoxStyles}
+                        />
+
+                        {contentType === 'Code Example' && (
+                            <ExtraCodeExampleCheckboxes {...filterProps} />
                         )}
-                        sx={spanAllColumns}
-                    />
+
+                        <SearchResults
+                            {...resultsProps}
+                            pageNumber={pageNumber}
+                            slug={topicSlug}
```

Comment:
Loading more on the topic page is resulting in a missing slash which could impact SEO
http://localhost:3000/developer/languages/javascript?page=2
Should be
http://localhost:3000/developer/languages/javascript/?page=2